rust/ql/lib/codeql/rust/dataflow/FlowSink.qll

```diff
@@ -1,4 +1,20 @@
-/** Provides classes and predicates for defining flow sinks. */
+/**
+ * Provides classes and predicates for defining flow sinks.
+ *
+ * Flow sinks defined here feed into data flow configurations as follows:
+ *
+ *   data from `*.model.yml` | QL extensions of `FlowSink::Range`
+ *    v                         v
+ *   `FlowSink` (associated with a models-as-data `kind` string)
+ *    v
+ *   `sinkNode` predicate | other QL defined sinks, for example using concepts
+ *    v                      v
+ *   various `Sink` classes for specific data flow configurations <- extending `QuerySink`
```

Comment:
This QL doc does not render well in VS code. Consider using a code block, e.g.
````
 * ```
 * data from `*.model.yml` or QL extensions of `FlowSink::Range`
 *   v
 * `FlowSink` (associated with a models-as-data `kind` string)
 *   v
 * `sinkNode` predicate | other QL defined sinks, for example using concepts
 *   v
 * various `Sink` classes for specific data flow configurations <- extending `QuerySink`
 * ```
````